actionview/lib/action_view/lookup_context.rb

```diff
@@ -67,7 +67,7 @@ class DetailsKey #:nodoc:
       def self.get(details)
         if details[:formats]
           details = details.dup
-          details[:formats] &= Mime::SET.symbols
+          details[:formats] &= Template::Types::Type.types.to_a
```

Comment:
Do we have to coerce to an array or is `types` returning an enumerable good enough?